src/middleware/header.js

```diff
@@ -105,44 +102,15 @@ middleware.renderHeader = async function renderHeader(req, res, data) {
 
 	templateValues.bootswatchSkin = (parseInt(meta.config.disableCustomUserSkins, 10) !== 1 ? res.locals.config.bootswatchSkin : '') || meta.config.bootswatchSkin || '';
 	templateValues.config.bootswatchSkin = templateValues.bootswatchSkin || 'noskin';	// TODO remove in v1.12.0+
-
-	const unreadCounts = results.unreadData.counts;
-	const unreadCount = {
-		topic: unreadCounts[''] || 0,
-		newTopic: unreadCounts.new || 0,
-		watchedTopic: unreadCounts.watched || 0,
-		unrepliedTopic: unreadCounts.unreplied || 0,
-		chat: results.unreadChatCount || 0,
-		notification: results.unreadNotificationCount || 0,
-	};
-
-	Object.keys(unreadCount).forEach(function (key) {
-		if (unreadCount[key] > 99) {
-			unreadCount[key] = '99+';
-		}
-	});
-
-	const tidsByFilter = results.unreadData.tidsByFilter;
-	results.navigation = results.navigation.map(function (item) {
-		function modifyNavItem(item, route, filter, content) {
-			if (item && validator.unescape(item.originalRoute) === route) {
-				unreadData[filter] = _.zipObject(tidsByFilter[filter], tidsByFilter[filter].map(() => true));
-				item.content = content;
-				if (unreadCounts[filter] > 0) {
-					item.iconClass += ' unread-count';
-				}
-			}
-		}
-		modifyNavItem(item, '/unread', '', unreadCount.topic);
-		modifyNavItem(item, '/unread?filter=new', 'new', unreadCount.newTopic);
-		modifyNavItem(item, '/unread?filter=watched', 'watched', unreadCount.watchedTopic);
-		modifyNavItem(item, '/unread?filter=unreplied', 'unreplied', unreadCount.unrepliedTopic);
-		return item;
-	});
-
 	templateValues.browserTitle = results.browserTitle;
-	templateValues.navigation = results.navigation;
-	templateValues.unreadCount = unreadCount;
```

Comment:
new code no longer setting `templateValues.unreadCount`?

Reply:
It is, just one line below :smile: